### PR TITLE
Change test url

### PR DESCRIPTION
**Pull request by wamesro**
Gopay changed test url
from `https://testgw.gopay.cz/` to `https://gw.sandbox.gopay.com/`

https://doc.gopay.com/cs/#dokumentace-gopay-rest-api

### CODE REVIEW

**Comment by wamesro** [comment]
https://doc.gopay.com/cs/#dokumentace-gopay-rest-api

**Comment by f3l1x** [comment]
@wamesro Thank you for the PR, but as you can see, our integration tools shows some fails.

https://travis-ci.org/Markette/GopayApi/jobs/223140982

**Comment by wamesro** [comment]
@f3l1x Sorry, but I changed only the addresses, I did not change the annotations.

File: src/GopayConfig.php [hunk 1]
```diff
@@ -11,10 +11,16 @@ class GopayConfig
...
+     * Testovaci a produkcni URL
+     */
+    const TEST_URL = 'https://gw.sandbox.gopay.com/';
+    const PROD_URL = 'https://gate.gopay.cz/';
+
 	/**
 	 * Parametr specifikujici, pracuje-li se na testovacim ci provoznim prostredi
 	 *
-	 * @var string
+	 * @var string $version verze
```
Comment (f3l1x) [inline src/GopayConfig.php]: This is not needed.

Comment (wamesro) [inline src/GopayConfig.php]: removed

**Comment by f3l1x** [comment]
Thank you.

**Pull request merged**